extensions/teams/package.json

```diff
@@ -24,15 +24,15 @@
     },
     "dependencies": {
         "botbuilder": "^4.22.1",
-        "restify": "^11.1.0"
+        "restify": "^10.0.0"
     },
     "devDependencies": {
-        "@types/node": "^18.19.26",
-        "@types/restify": "^8.5.12",
+        "@types/node": "^18.0.0",
```

Comment:
Can we leave this as `^18.19.26`? I believe the outcome will be the same, but dependabot will just try and bump this version if we lower it. Same for `@types/restify` and `ts-node`.

Reply:
Yeah, that is fine.

Reply:
I bumped this to 18.20.1 since that is what I am testing with.